### PR TITLE
Set default PAGE_SIZE for Api in __init__ (#64)

### PR DESCRIPTION
**Pull request by jcampbell**
Sets the default value of PAGE_SIZE for the Api object when it is initialized using a Flask app (as it is in current documentation examples).

### CODE REVIEW

**Comment by akira-dev** [comment]
I can't merge this PR until continuous integration test fails. Please fix tests if you want to include this fix soon.

**Comment by jpcampb2** [comment]
I adjusted Api's __init__ to create a default Flask object if no app is provided.

File: flask_rest_jsonapi/api.py [hunk 1]
```diff
@@ -20,6 +21,10 @@ def __init__(self, app=None, blueprint=None, decorators=None):
         :param blueprint: a flask blueprint
         :param tuple decorators: a tuple of decorators plugged to each resource methods
         """
+
+        if app is None:
+            app = Flask(__name__)
```
Comment (hellupline) [inline flask_rest_jsonapi/api.py]: I dont think this is a good idea:

If the user create a instance of the jsonapi, then configure it using .init_app(app):
the user would have created a second Flask app, 

instead, __init__ shoud try to call init_app if app is not None.

**Comment by jcampbell** [comment]
Thanks, @hellupline. Closing this as irrelevant now based on your new pull request.

**Pull request closed without merging**